### PR TITLE
Simplify nextjs example

### PR DESCRIPTION
**Pull request by paul-sachs**
- Splitting example into streaming and unary example
- Simplify message sending via form submit

### CODE REVIEW

File: nextjs/pages/index.tsx [hunk 1]
```diff
@@ -1,118 +1,90 @@
...
-                <Link href="/ssr">View SSR Example</Link>
+                <h4 className={styles.subtitle}>
+                    Next.js + Client-side Fetching
+                </h4>
+                <div className={styles.links}>
+                    <Link href="/ssr">View SSR Example</Link>
+                    <Link href="/server-streaming">
+                        View Server Streaming Example
+                    </Link>
+                </div>
```
Comment (timostamm) [inline nextjs/pages/index.tsx:56]: Can we change the header to provide a better navigation?

Screenshot 2023-06-23 at 17 54 50

**Review by timostamm — COMMENTED** [review]
Looks great to me, thank you!

File: nextjs/pages/index.tsx (hunk 1 above)
Comment (paul-sachs) [reply nextjs/pages/index.tsx:56]: Yeah i'll make those changes, makes sense.

Comment (paul-sachs) [reply nextjs/pages/index.tsx]: Sorry for the delay. Updated!

**Comment by smaye81** [comment]
I was going to say let's change to the `connectrpc` paths here, but let's just merge this and then we can update them all in one PR (#777).

**Review by smaye81 — APPROVED** [review]

**Pull request merged**